blog/2024/10-26-jellyfin-release-10.10.0/index.mdx

```diff
@@ -0,0 +1,80 @@
+
+Happy watching!
+
+\- Joshua
+
+{/* truncate */}
+
+## Breaking Changes
+
+* Most clients should continue to work as-is without any issues or any forced upgrades, though this may change in the future. The sole exception is Jellyfin4Kodi, which currently has issues due to point 5 below, though a fix is hopefully forthcoming shortly.
+
+* We have deprecated Raspberry Pi V4L2 hardware transcoding support, and are looking to deprecate 32-bit ARM support (i.e. `armv7` or `armhf`) in general for 10.10.0 and later, with a goal to remove 32-bit ARM support in 10.11.0. Older, slow ARM systems have always been tricky platforms for us: they're popular but extremely poor in performance, and this has caused a lot of people a lot of headaches. With the RPi5 entirely removing a hardware acceleration engine, and most ARM single-board computers now supporting ARM64, we are taking this action now to ensure it's widely known.  If you are running Jellyfin on old ARM hardware, now is the time to start [looking at an upgrade](https://jellyfin.org/docs/general/administration/hardware-selection). See [PR #1148](https://github.com/jellyfin/jellyfin.org/pull/1148).
+
+* We now use the system temporary storage engine (e.g. `/tmp` on Linux) for storing short-lived transient temporary files, to allow us to leverage temporary ramdisks and the like and avoid cluttering potentially slow storage. This may cause issues if you specifically depended on the previous behaviour. Note that this does not include transcoding temporary files. See [PR #12226](https://github.com/jellyfin/jellyfin/pull/12226).
+
+* The server will now refuse to start if `ffmpeg` cannot be found, is an incorrect version, or does not function properly (missing extensions, etc.). With how critical `ffmpeg` is to Jellyfin, this has become very important to avoid mis-reported issues. This can be explicitly bypassed if needed. See [PR #12463](https://github.com/jellyfin/jellyfin/pull/12463).
+
+* Network paths in libraries have been fully removed and will no longer work. This functionality has been deprecated for a long time, and most of it was removed in 10.9.0, but this removes the remainder. See [PR #12446](https://github.com/jellyfin/jellyfin/pull/12446). Third-party clients relying on this functionality should be able to re-implement it as required.
+
+* Systems with Trickplay enabled may see a relatively long database migration occurring during the upgrade. If Jellyfin seems to hang starting up after upgrading, please observe the logs and wait for the migration to complete. This helps enable the functionality below.
```

Comment:
```suggestion
* Systems with Trickplay enabled may see a relatively long migration occurring during the upgrade. If Jellyfin seems to hang starting up after upgrading, please observe the logs and wait for the migration to complete. This helps enable the functionality below.
```

It isn't a database migration

Reply:
Good call, wrote that before understanding what it was.